python-pytest.el

```diff
@@ -329,6 +329,13 @@ With a prefix ARG, allow editing."
   "Major mode for pytest sessions (derived from comint-mode)."
   (compilation-setup))
 
+(defvar python-pytest-finished-mode-map
+  (let ((map (make-sparse-keymap)))
+    (set-keymap-parent map compilation-minor-mode-map)
+    (define-key map "g" #'python-pytest-repeat)
```

Comment:
maybe this should be a "remap" key binding?

Reply:
Nice catch!